output/typescript/types.ts

```diff
@@ -139,7 +143,7 @@ export interface ExtendedStatsAggregate extends StatsAggregate {
 }
 
 export interface FiltersAggregate extends AggregateBase {
-  buckets: Array<FiltersBucketItem>
+  buckets: Array<FiltersBucketItem> | Record<string, FiltersBucketItem>
```

Comment:
Should we use `Record` or `Dictionary`?

Reply:
The input is `Dictionary` but the output is always `Record`

Reply:
My bad, should have looked at the file name!
Some times this output is hidden, others not 😅